### PR TITLE
set proper token precision

### PR DESCRIPTION
**Pull request by askmike**

### CODE REVIEW

File: contracts/Ico.sol [hunk 1]
```diff
@@ -15,17 +15,15 @@ contract Ico is BasicToken {
   uint8 public constant decimals = 18;
 
   // Significant digits tokenPrecision
-  uint256 private tokenPrecision = 10e18;
+  uint256 private tokenPrecision = 10e17;
 
   // TODO: set this final, this equates to an amount
   // in dollars.
-  uint256 public HARD_CAP = 10000 * tokenPrecision;
+  uint256 public hardCap = 10000 * tokenPrecision;
```
Comment (Evelyn-McDonald) [inline contracts/Ico.sol]: Did you find the answer to the case sensitivity? Do the variables actually behave in a different way?

**Comment by askmike** [comment]
Didn't have an impact, but forgot to revert it. Though camelCase is more
consistent with the rest of the codebase.

File: package.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
   "license": "MIT",
   "dependencies": {
     "moment": "^2.19.3",
+    "web3": "^1.0.0-beta.26",
```
[the remark this answers is not in the document]
Comment (askmike) [reply package.json]: woops, no we don't. Removed it.

**Comment by ale316** [comment]
Should be good to merge

**Comment by Evelyn-McDonald** [comment]
I'm cool with merging this

**Pull request merged**